vpc_peering_connection.go

```diff
@@ -16,13 +16,15 @@ type (
 	// CreateVPCPeeringConnectionRequest holds the parameters to create a new
 	// peering connection.
 	CreateVPCPeeringConnectionRequest struct {
-		PeerCloudAccount string `json:"peer_cloud_account"`
-		PeerVPC          string `json:"peer_vpc"`
+		PeerCloudAccount string  `json:"peer_cloud_account"`
+		PeerVPC          string  `json:"peer_vpc"`
+		PeerRegion       *string `json:"peer_region,omitempty"`
 	}
 )
 
-// Create the given VPC on Aiven.
-func (h *VPCPeeringConnectionsHandler) Create(
+// CreateCrossRegion the given VPC on Aiven.
+// NOTE! PeerRegion == nil creates the peering within the same region
+func (h *VPCPeeringConnectionsHandler) CreateCrossRegion(
```

Comment:
Is this new method actually needed? What doesn't work if the existing Create is just used always?

Reply:
true; changed